docs/corpora_schema.md

```diff
@@ -74,24 +80,21 @@ This is usually the case when there are many ways to produce the matrix layer in
 
 ### Integration Metadata
 
-#### Cell Metadata
-
-To support data integration, each cell MUST have the following metadata:
+cellxgene requires ontology terms to enable search, comparison, and integration of data.
+Ontology terms MUST use [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning they are a CURIE where the prefix identifies the ontology.
+For example `EFO:0000001` is a term in the `EFO` ontology.
 
-**Field name**|**Constraints**
-:--|:--
-tissue|string. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
-assay|string
-disease|string
-cell\_type|string
-sex|"male", "female", "mixed", "unknown", or "other"
-ethnicity|string, "na" if non-human, "unknown" if not available
-development\_stage|string, "unknown" if not available
+When no appropriate ontology value is available, then the most precise accurate term MUST be used.
+For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL ontology is CL:0000099 ("Interneuron"), then the interneuron term can be used to fulfill this requirement, and ensures that users searching for "neuron" are able to find these data.
+Users will still be able to access more specific cell type annotations that have been submitted with the data (but aren't required by the schema).
+A dataset comprising cells of the human embryo provides a more extreme example.
+In this case, the most  precise accurate term may be the root of the cell ontology `cell`, or its child term `cell in vitro`.
+The Cell Ontology is expanding over time, and we hope to migrate datasets to more defined terms as they are defined.
+In the mean time, having these less precise terms maximizes the findability (and therefore reusability) of datasets.
```

Comment:
Could be reading this incorrectly but here you say "having .. less precise terms maximizes findability" but we ask them to use "the most precise accurate term"?

Reply:
This wording could be tightened up, thanks for pointing this out. The "less precise terms" refer to the "most precise accurate cell ontology terms" that nevertheless don't describe the cell types as well as the author's novel term. 

I'll go with: _"having the *cell ontology* terms maximizes the findability (and therefore reusability) of datasets"_